docs/overview/getting-started/intro.md

```diff
@@ -2,69 +2,53 @@
 slug: /intro
 ---
 
-# Overview
+# Introduction
 
-## Introduction
+Avalanche is an open-source platform for building decentralized applications (Dapps) in one
+interoperable, highly scalable ecosystem. Avalanche is the first ecosystem designed to accommodate
+the scale of global finance, with near-instant transaction finality.
 
-[Avalanche](https://avax.network) is an open-source platform for launching [decentralized
-applications](https://support.avalabs.org/en/articles/4587146-what-is-a-decentralized-application-dapp)
-and enterprise [blockchain](http://support.avalabs.org/en/articles/4064677-what-is-a-blockchain)
-deployments in one interoperable, highly scalable ecosystem. Avalanche is the first decentralized
-smart contracts platform built for the scale of global finance, with near-instant transaction
-finality. Ethereum developers can quickly build on Avalanche as Solidity works out-of-the-box.
+Avalanche empowers developers to join a rich ecosystem of blockchains known as Subnets. Subnets
+are isolated clusters of the Avalanche network that allow any Dapp to run on its own dedicated set
+of nodes, enabling unmatched levels of fault-tolerance and customization for any use-case. Each
+Subnet is isolated from other Subnets in the ecosystem, so increased usage on a single Subnet 
+doesn't affect other Subnets on Avalanche.
 
-A key difference between Avalanche and other decentralized networks is the consensus protocol. Over
-time, people have come to a false understanding that blockchains have to be slow and not scalable.
-The Avalanche protocol employs a novel approach to consensus to achieve its strong safety
-guarantees, quick finality, and high-throughput without compromising decentralization.
+In Avalanche, developers can create their Dapps by implementing them as a Virtual Machine (VM).
```

Comment:
I think this paragraph makes it sound a bit like developers _need_ to build virtual machines to write dapps. I would emphasize that that is something we support but is not required. For example: developers can deploy their own Solidity smart contracts or reinvent the smart contract paradigm by building custom virtual machines. Or something like that

Reply:
Good call. I reworded this a bit.